vcpkg.json

```diff
@@ -0,0 +1,18 @@
+{
+  "dependencies": [
+    "curl",
+    "freetype",
+    "gettext",
+    "gmp",
+    "jsoncpp",
+    "libjpeg-turbo",
+    "libogg",
+    "libvorbis",
```

Comment:
i think it would make sense to include all optional depends.
right now it's inconsistent (sound support, but no SSL)

Reply:
Makes sense, let me update this list and rebuild